autokeras/auto_model.py

```diff
@@ -311,3 +311,13 @@ def evaluate(self, x, y=None, batch_size=32, **kwargs):
         data = preprocess_graph.preprocess(
             self._process_xy(x, y))[0].batch(batch_size)
         return model.evaluate(data, **kwargs)
+
+    def export_keras_model(self):
+        """Export the best Keras Model.
+
+        # Returns
+            tf.keras.Model. The best model found during the search with trained
```

Comment:
"`tf.keras.Model` instance. The best model found during the search, loaded with trained weights."